resources/views/omnomcom/withdrawals/show.blade.php

```diff
@@ -130,73 +132,77 @@ class="btn btn-outline-success btn-block">
+                                        @endisset
+                                    @endif
+                                    <td>{{ $data->count }}</td>
+                                    <td>&euro;{{ number_format($data->sum, 2, ',', '.') }}</td>
+                                    @if(!$withdrawal->closed)
+                                        <td>
+                                            @if($withdrawal->getFailedWithdrawal($data->user))
+                                                Failed
+                                                @include('components.modals.confirm-modal', [
+                                                   'action' => route('omnomcom::orders::delete', ['id'=>$withdrawal->getFailedWithdrawal($data->user)->correction_orderline_id]),
+                                                   'text' => '(Revert)',
+                                                   'title' => 'Confirm Revert',
+                                                   'message' => 'Are you sure you want to revert this withdrawal? The user will <b>NOT</b> automatically receive an e-mail about this!',
+                                                   'confirm' => 'Revert',
+                                                ])
+                                            @else
+                                                @include('components.modals.confirm-modal', [
+                                                   'action' => route('omnomcom::withdrawal::markloss', ['id' => $withdrawal->id, 'user_id' => $data->user->id]),
+                                                   'text' => 'Loss',
+                                                   'title' => 'Confirm Marking Loss',
+                                                   'message' => 'Are you sure you want to mark this withdrawal for '.$data->user->name.' as a loss? <b>This cannot easily be undone!</b>',
+                                                   'classes' => 'text-white fw-bold underline-on-hover'
+                                                ])
+                                            @endif
+                                        </td>
+                                    @endif
+                                </tr>
 
                         @endforeach
-
                     </table>
+                    @if(!$withdrawal->closed)
+                        <div class="card-footer bg-dark text-white mb-2 mr-2">
+                            <div class="d-inline-flex flex-row flex-row-reverse w-100">
+                                <button name="action" type="submit" value="remove" class="btn btn-danger">
+                                    Remove for selected users
+                                </button>
+
+                                <button name="action" type="submit" value="markfailed" class="btn btn-warning me-2">
+                                    Mark failed for selected users
+                                </button>
```

Comment:
The single update buttons implement a confirm modal for this action. I think this should also be required for the bulk update action, as it is even more impactful.

Reply:
I removed them because when you do select orderlines and then remove one orderline your selection will be gone. I think only having one way is more clear.

Reply:
Yes I agree, I do not understand the confirm-modal enough yet to implement this though